netcontrold/tests/lib/test_util.py

```diff
@@ -262,3 +262,35 @@ def test_rr_cpu_in_numa_positive(self, mock_open):
         out = util.rr_cpu_in_numa()
         expected = [0, 2, 1, 3]
         self.assertEqual(out, expected)
+
+
+class TestUtil_test_thread_creation(TestCase):
+
+    def setUp(self):
+        util.Memoize.forgot = True
+        thread = util.Thread()
+
+    def check_thread_exist(thread):
+        if threading.activeCount() > 0:
+            value1 = 'T'
+        else:
+            value1 = 'F'
+
+        count = util.Thread_status()
+        if count > 0:
+            value2 = 'T'
+        else:
+            value2 = 'F'
+
+        self.assertEqual(value1, value2)
```

Comment:
Although you will rewrite test, just fyi the flaw in the above assert: when both are 'F', the test will still pass.